src/software/ai/hl/stp/tactic/chip_tactic_test.cpp

```diff
@@ -2,15 +2,292 @@
 
 #include <gtest/gtest.h>
 
+#include "software/ai/hl/stp/action/chip_action.h"
 #include "software/test_util/test_util.h"
 
-TEST(ChipTacticTest, constructor_test)
+bool compareChipActions(ChipAction *chip_action, ChipAction *expected_chip_action) {
```

Comment:
We should take `shared_ptr<ChipAction>` as parameters, since we generally try avoid raw pointers in our code.

Reply:
Rewritten to use `shared_ptr<ChipAction>`